hphp/test/zend/good/ext/standard/tests/strings/strncmp_variation6.php.expectf

```diff
@@ -771,6 +771,6 @@ int(0)
 int(0)
 
 -- Checking with out of character's range, given in binary format --
-int(1)
+bool(true)
```

Comment:
This also needs to be handled in the script.  Sorry didn't notice this last time.  See something like https://github.com/swalk-cavium/hhvm/blob/1b3401bb9ccbbd2dd5f2c61059270ea54ece9c95/hphp/test/tools/import_zend_test.py#L1381